pkgs/applications/audio/aaxtomp3/default.nix

```
@@ -0,0 +1,55 @@
+{ coreutils
+, fetchFromGitHub
+, ffmpeg
+, findutils
+, gnugrep
+, gnused
+, jq
+, lame
+, lib
+, makeWrapper
+, mediainfo
+, mp4v2
+, stdenv
+}:
+stdenv.mkDerivation rec {
+  pname = "aaxtomp3";
+  version = "1.3";
+
+  src = fetchFromGitHub {
+    owner = "krumpetpirate";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-7a9ZVvobWH/gPxa3cFiPL+vlu8h1Dxtcq0trm3HzlQg=";
+  };
+
+  doBuild = false;
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  runtimeInputs = [
+    coreutils
+    ffmpeg
+    findutils
+    gnugrep
+    gnused
+    jq
+    lame
+    mediainfo
+    mp4v2
+  ];
+
+  installPhase = ''
+    install -Dm755 AAXtoMP3 $out/bin/aaxtomp3
+    wrapProgram $out/bin/aaxtomp3 --prefix PATH : ${lib.makeBinPath runtimeInputs}
```

```suggestion

  installPhase = ''
    install -Dm755 AAXtoMP3 $out/bin/aaxtomp3
    wrapProgram $out/bin/aaxtomp3 --prefix PATH : ${lib.makeBinPath [ coreutils
      ffmpeg
      findutils
      gnugrep
      gnused
      jq
      lame
      mediainfo
      mp4v2 
    ]}
```
Are those all strict requirements? Are some optionals we could drop?

---

all of these binaries are called within the `aaxtomp3` script, confirmed via manual review, and the author does not list any as optional or critical. we also do not have any way to remove logic other than patching the source, so there is the possibility that the user could end up with a broken script.

what is standard in nix for what other distros would call an "optional dependency"?

---

Okay but we shouldn't use runtimeInputs here because that is not used with mkDerivation or wrapProgram but some other builders use it which can be confusing.

---

makes sense; would you prefer a different name, let variable, or something else?